index.html

```diff
@@ -5058,39 +5068,47 @@ <h3 id="playback-AudioBufferSourceNode">
             subsequent sample frame in <code>buffer</code>, then interpolation
             should be based on the sequence of subsequent frames beginning at
-            <code>actualLoopStart</code>.
-            </li>
-            <li>Let <code>bufferTime</code> equal 0. This is the <em>playhead
-            position</em> within <code>buffer</code> of the next output sample
-            frame. The initial value -1 indicates that this position has not
-            yet been determined.
+            <code>loopStart</code>.
             </li>
-            <li>Let <code>started</code> equal <code>false</code>, indicating
-            whether output of the buffer contents has begun or not.
-            </li>
-            <li>For each block of audio to be rendered:
+            <li>For each quantum of audio to be rendered:
               <ol>
-                <li>Let <code>currentTime</code> be the value of
-                <code>currentTime</code> attribute of the node's
-                <a>AudioContext</a>.
-                </li>
-                <li>Let <code>sampleRate</code> be the value of the
-                <code>sampleRate</code> attribute of the node's
-                <a>AudioContext</a>.
-                </li>
-                <li>Let <code>dt</code> be <code>1 / sampleRate</code>.
-                </li>
-                <li>Let <code>index</code> be 0.
-                </li>
-                <li>Let <code>enteredLoop</code> be <code>false</code>.
+                <li>Initialize state for this render quantum:
+                  <ol>
+                    <li>Let <code>currentTime</code> be the value of
+                    <code>currentTime</code> attribute of the node's
+                    <a>AudioContext</a>.
+                    </li>
+                    <li>Let <code>sampleRate</code> be the value of the <code>
+                      sampleRate</code> attribute of the node's
+                      <a>AudioContext</a>.
+                    </li>
+                    <li>Let <code>dt</code> be <code>1 / sampleRate</code>.
+                    </li>
+                    <li>Let <code>index</code> be 0.
+                    </li>
+                    <li>Let <code>computedPlaybackRate</code> be
+                    <code>playbackRate * pow(2, detune / 1200)</code> . This
+                    combines the two k-rate parameters affecting playback rate
+                    into a single computed quantity.
+                    </li>
+                  </ol>
                 </li>
-                <li>Let <code>computedPlaybackRate</code> be <code>playbackRate
-                * pow(2, detune / 1200)</code> . This combines the two k-rate
-                parameters affecting playback rate into a single computed
-                quantity.
+                <li>If <code>loop</code> is true,
+                  <ol>
+                    <li>If <code>loopStart</code> &gt; 0 and
```

Comment:
This evaluation is actually k-rate, because it takes place outside the per-frame loop (but inside the per-quantum loop).

This illustrates the reason why I think that describing both loops is important in the algorithm.